src/config.js

```diff
@@ -335,6 +343,20 @@ export const runtimeConfig =
         CASHAPP: nodeIsProduction
           ? process.env.CASHAPP
           : process.env.RAZZLE_CASHAPP,
+        TEESPRING: nodeIsProduction
```

Comment:
I think these need to be:

```javascript
TEESPRING: nodeIsProduction ?  ? process.env.TEESPRING
          : process.env.RAZZLE_TEESPRING
```

basically use a different variable if not in production, otherwise this looks great!!!